src/components/calendar/components/EventForm.tsx

```
@@ -0,0 +1,128 @@
+                onChange={e => setNewEvent(prev => ({ ...prev, endTime: e.target.value }))}
+                required
+              />
+            </div>
+          </div>
+          <div className="space-y-2">
+            <Label htmlFor="eventType">Event Type</Label>
+            <select
+              id="eventType"
+              className="flex h-10 w-full rounded-md border border-input bg-background px-3 py-2 text-sm ring-offset-background focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2"
+              value={newEvent.type}
+              onChange={e => setNewEvent(prev => ({ ...prev, type: e.target.value as any }))}
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)